.docker/Dockerfile.kimai-base

```diff
@@ -0,0 +1,210 @@
+ARG TIMEZONE="Europe/London"
+
+###########################
+# Shared tools
+###########################
+
+# composer base image
+FROM composer:${COMPOSER_VER} AS composer
+
+###########################
+# PHP extensions
+###########################
+
+#fpm alpine php extension base
+FROM php:${PHP_VER}-fpm-alpine AS fpm-php-ext-base
+RUN apk add --no-cache \
+    # build-tools
+    autoconf \
+    dpkg \
+    dpkg-dev \
+    file \
+    g++ \
+    gcc \
+    icu-dev \
+    libatomic \
+    libc-dev \
+    libgomp \
+    libmagic \
+    m4 \
+    make \
+    mpc1 \
+    mpfr4 \
+    musl-dev \
+    perl \
+    re2c \
+    # gd
+    freetype-dev \
+    libpng-dev \
+    # icu
+    icu-dev \
+    icu-data-full \
+    # ldap
+    openldap-dev \
+    libldap \
+    # zip
+    libzip-dev \
+    # xsl
+    libxslt-dev
+
+
+# apache debian php extension base
+FROM php:${PHP_VER}-apache-bookworm AS apache-php-ext-base
+RUN apt-get update
+RUN apt-get install -y \
+        libldap2-dev \
+        libicu-dev \
+        libpng-dev \
+        libzip-dev \
+        libxslt1-dev \
+        libfreetype6-dev
+
+
+# php extension gd - 13.86s
+FROM ${BASE}-php-ext-base AS php-ext-gd
+RUN docker-php-ext-configure gd \
+        --with-freetype && \
+    docker-php-ext-install -j$(nproc) gd
+
+# php extension intl : 15.26s
+FROM ${BASE}-php-ext-base AS php-ext-intl
+RUN docker-php-ext-install -j$(nproc) intl
+
+# php extension ldap : 8.45s
+FROM ${BASE}-php-ext-base AS php-ext-ldap
+RUN docker-php-ext-configure ldap && \
+    docker-php-ext-install -j$(nproc) ldap
+
+# php extension pdo_mysql : 6.14s
+FROM ${BASE}-php-ext-base AS php-ext-pdo_mysql
+RUN docker-php-ext-install -j$(nproc) pdo_mysql
+
+# php extension zip : 8.18s
+FROM ${BASE}-php-ext-base AS php-ext-zip
+RUN docker-php-ext-install -j$(nproc) zip
+
+# php extension xsl : ?.?? s
+FROM ${BASE}-php-ext-base AS php-ext-xsl
+RUN docker-php-ext-install -j$(nproc) xsl
+
+# php extension redis
+FROM ${BASE}-php-ext-base AS php-ext-redis
+RUN yes no | pecl install redis && \
+    docker-php-ext-enable redis
+
+# php extension opcache
+FROM ${BASE}-php-ext-base AS php-ext-opcache
+RUN docker-php-ext-install -j$(nproc) opcache
+
+###########################
+# fpm base build
+###########################
+
+# fpm base build
+FROM php:${PHP_VER}-fpm-alpine AS fpm-base
+ARG TIMEZONE
+RUN apk add --no-cache \
+        bash \
+        coreutils \
+        freetype \
+        haveged \
+        icu \
+        icu-data-full \
+        libldap \
+        libpng \
+        libzip \
+        libxslt-dev \
+        fcgi \
+        tzdata && \
+    touch /use_fpm
+
+EXPOSE 9000
+
+HEALTHCHECK --interval=20s --timeout=10s --retries=3 \
+    CMD \
+    SCRIPT_NAME=/ping \
+    SCRIPT_FILENAME=/ping \
+    REQUEST_METHOD=GET \
+    cgi-fcgi -bind -connect 127.0.0.1:9000 || exit 1
+
+
+
+###########################
+# apache base build
+###########################
+
+FROM php:${PHP_VER}-apache-bookworm AS apache-base
+ARG KIMAI
```

Comment:
This ARG should be removed otherwise the following warning is raised during the build:
`WARN[0000] missing "KIMAI" build argument. Try adding "--build-arg KIMAI=<VALUE>" to the command line`